src/screens/authentication/Login/index.js

```diff
@@ -6,9 +6,10 @@ import * as AuthenticationService from '~/authenticationService'
 import Auth from '../components/Auth'
 import { isValidEmail } from '../utils'
 
-const Login = () => {
+const Login = ({ navigation }) => {
   const [authenticating, setAuthenticating] = useState(false)
   const [error, setError] = useState(null)
+  const emailSent = navigation.getParam('emailSent')
```

Comment:
I think the name of this navigation param is not descriptive enough. Which email was sent? Maybe change it to `passwordRecoveryEmailSent`